mindmeld/system_entity_recognizer.py

```diff
@@ -55,6 +55,7 @@ class SystemEntityError(Exception):
 
 class DucklingDimension(Enum):
     AMOUNT_OF_MONEY = "amount-of-money"
+    CREDIT_CARD_NUMBER = "credit-card-number"
```

Comment:
If Duckling can now recognize credit card numbers, we can add this functionality to the auto-annotator as well by adding "sys_credit_card_number" to the DUCKLING_TO_SYS_ENTITY_MAPPINGS for the languages that it is supported for. However, since Spacy doesn't recognize this entity, it would need to be used in the setting where the largest duckling candidate spans are labelled. May be a feature with a limited use case at this time, however, it could be potentially be supported in future PR.